x-pack/test/api_integration/apis/aiops/example_stream.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
Could remove the duplicate license header.

Reply:
Thanks for spotting this! Fixed in [4204428](https://github.com/elastic/kibana/pull/131317/commits/42044289894da091441713bb8aadc8f03f5da992).